Horizon/Horizon/Sources/Features/ModuleItemSequence/View/FileDetails/Views/FileDetailsViewModel.swift

```diff
@@ -26,10 +26,8 @@ final class FileDetailsViewModel {
     // MARK: - Private Properties
 
     private var subscription: AnyCancellable?
+    private var viewController: WeakViewController = WeakViewController()
```

Comment:
We don't need to store this view controller. Since it's now passed to the `downloadFile` function, we can pass it forward to `showShareSheet`.